src/core/annotation.js

```diff
@@ -297,6 +297,7 @@ class AnnotationFactory {
             baseFont.set("Type", Name.get("Font"));
             baseFont.set("Subtype", Name.get("Type1"));
             baseFont.set("Encoding", Name.get("WinAnsiEncoding"));
+            baseFont.set("Name", Name.get("Helv"));
```

Comment:
How are these "Helv" changes related to the patch, since the commit message says nothing about that :-P